recipes/e3sm-unified/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+mpi:
+  - nompi
+  - openmpi
+  - mpich
+  - hpc
```

Comment:
I'm aware that this isn't a standard MPI type on conda-forge.  It's useful for us because we skip certain packages in the metapackage that we instead intend to build using system MPI on our HPC machines.